lib/PuppeteerSharp/ElementHandle.cs

```diff
@@ -384,9 +384,9 @@ public async Task<BoxModel> BoxModelAsync()
         /// <returns>Resolves to the content frame</returns>
         public async Task<Frame> ContentFrameAsync()
         {
-            var nodeInfo = await Client.SendAsync<DomDescribeNodeResponse>("DOM.describeNode", new
+            var nodeInfo = await Client.SendAsync<DomDescribeNodeResponse>("DOM.describeNode", new Dictionary<string, object>
```

Comment:
Why are we changing this to a dictionary?

Reply:
I noticed that the way the parameter for this was not correct as the docs specify multiple parameters.

Correcting it seemed like the correct thing to do, admittedly not related to the PR focus.

https://chromedevtools.github.io/devtools-protocol/tot/DOM#method-describeNode